buster/utils.py

```diff
@@ -27,9 +25,7 @@ def download_db(db_url: str, output_dir: str):
 def get_documents_manager_from_extension(filepath: str) -> Type[DocumentsManager]:
```

Comment:
I'm wondering if we can safely remove those functions now

Reply:
yes we could, might as well do it in this PR